Controllers/ApplicationController.cs

```diff
@@ -134,8 +134,11 @@ private static Member InitializeMember(TokenClient tokenClient)
             // The Token UI shows this (and the alias) to the user when requesting access.
             member.SetProfile(new Profile
             {
-                DisplayNameFirst = "PFM Demo"
+                DisplayNameFirst = "Demo",
```

Comment:
Can we set first name only and use "Demo PFM"? Don't need to separate the name into "first" and "last" in this case.